### PR TITLE
👷(circle) fix & improve bootstrap test

### PR DESCRIPTION
**Pull request by cchaudier**
## Purpose

Bootstrap deployment fails.

## Proposal

- [x] add a `host` configuration variable for the `hello` app to make it deployable
- [x] test the deployed app is responding to HTTP requests (http code + response)

### CODE REVIEW

**Review by jmaupetit — COMMENTED** [review]
`ci` should be an environment not a customer, right?

**Comment by jmaupetit** [comment]
Note that in future PR series, I will add jobs in the CI to test the deployment of all distributed apps.

**Comment by cchaudier** [comment]
@jmaupetit you're right CI is an env_type.
I've change this.
And yes we must curl the next routes to validate the app

**Comment by jmaupetit** [comment]
> And yes we must curl the next routes to validate the app

Will you add this to this PR?

File: .circleci/config.yml [hunk 1]
```diff
@@ -156,12 +157,20 @@ jobs:
...
+            OPENSHIFT_DOMAIN=$(hostname -I | awk '{print $1}')
+            oc cluster up --server-loglevel=5 --public-hostname="${OPENSHIFT_DOMAIN}"
+            oc login -u developer -p developer https://${OPENSHIFT_DOMAIN}:8443 --insecure-skip-tls-verify=true
             echo "arnold" | USE_TTY=false ./bin/bootstrap -e "env_type=ci"
-            sleep 10
-            wget --no-check-certificate "https://hello.patient0-ci.${LOCALE_IP}.nip.io"
+            for try in $(seq 5); do
+              echo "Testing hello application http response ($try)"
+              curl -vk "https://next.hello.ci-patient0.${OPENSHIFT_DOMAIN}.nip.io/" 2> /tmp/hello.err > /tmp/hello.out
+              if grep "HTTP/1.1 200 OK" /tmp/hello.err && grep "Hello OpenShift! by Arnold" /tmp/hello.out; then
```
Comment (sampaccoud) [inline .circleci/config.yml]: why would the 200 code response be written to stderr?

Comment (jmaupetit) [reply .circleci/config.yml]: In verbose mode, `curl` displays headers, response status etc in `stderr` and the http response content in `stdout`... and it works :tada:

**Review by sampaccoud — APPROVED** [review]

File: .circleci/config.yml (hunk 1 above)
Comment (cchaudier) [reply .circleci/config.yml]: :+1:

**Pull request merged**